go-controller/pkg/util/sriovnet_linux.go

```diff
@@ -56,6 +57,23 @@ func (defaultSriovnetOps) GetVfIndexByPciAddress(vfPciAddress string) (int, erro
 	return sriovnet.GetVfIndexByPciAddress(vfPciAddress)
 }
 
+func (defaultSriovnetOps) GetPfIndexByVfPciAddress(vfPciAddress string) (int, error) {
```

Comment:
@moshe010 does it worth to have this function in sriovnet library?

Reply:
https://github.com/k8snetworkplumbingwg/sriovnet/pull/64

I'm waiting for this to be merged. I will fix this PR once that happens.